examples/eks/eks_cluster_gitops/README.md

```diff
@@ -0,0 +1,26 @@
+## EKS and CAST AI example for GitOps onboarding flow
+
+Following example shows how to onboard EKS cluster to CAST AI using GitOps flow.
```

Comment:
Could you add/make clear for the reader that this example is a way to manage Node Configuration, Node Templates, etc using terraform, but all castware component need to be installed manually by other means.